.buildkite/pipelines/run_qa_tests.yml.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License
+# 2.0 and the following additional limitation. Functionality enabled by the
+# files subject to the Elastic License 2.0 may only be used in production when
+# invoked by an Elasticsearch process with a license key installed that permits
+# use of machine learning features. You may not use this file except in
+# compliance with the Elastic License 2.0 and the foregoing additional
+# limitation.
+
+cat <<EOL
+steps:
+  - label: "Appex QA Tests :test_tube:"
+    key: "appex_qa_tests"
+    command: 
+      - "sudo apt-get -y install openjdk-17-jre openjdk-17-jdk"
+      - 'buildkite-agent artifact download "build/*" . --step build_test_linux-aarch64-RelWithDebInfo'
+      - '.buildkite/scripts/steps/run_qa_tests.sh || (cd ../elasticsearch && find x-pack -name logs | xargs tar cvzf logs.tgz && buildkite-agent artifact upload logs.tgz && false)'
+    depends_on: "build_test_linux-x86_64-RelWithDebInfo"
```

Comment:
Why do we download the build artifacts for ARM but depend on Intel?

If we only run this on one platform I'd prefer it was Intel, because that's what we've chosen to run on in our production Cloud.